pandas/core/series.py

```diff
@@ -1372,6 +1379,8 @@ def reset_index(self, level=None, drop=False, name=lib.no_default, inplace=False
             when `drop` is True.
         inplace : bool, default False
             Modify the Series in place (do not create a new object).
+        allow_duplicates : bool, default False
+            Allow duplicate column labels to be created.
```

Comment:
need a versionadded anytime you have changed the signature

Reply:
Done